test/libsolidity/syntaxTests/iceRegressionTests/oversized_var.sol

```diff
@@ -0,0 +1,16 @@
+contract b {
+    struct c {
+        uint [ 2 ** 253 ] a ;
+    }
+
+    c d ;
+    function e () {
+        var d = d ;
+    }
+}
+// ----
+// SyntaxError 4937: (79-120): No visibility specified. Did you intend to add "public"?
```

Comment:
Can you simplify the test a little to only contain the relevant errors / warnings?

Reply:
Yeah - ICE regression tests are always a bit weird, so I usually don't bother too much, but this one can really go, that's true.

Reply:
All the others are actually required to trigger this, though, I think.